omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/libraries/omise-php/lib/Omise.php';
+    require_once _PS_MODULE_DIR_ . '/omise/libraries/omise-plugin/Omise.php';
+    require_once _PS_MODULE_DIR_ . '/omise/setting.php';
+}
+
+class Charge extends ModuleFrontController
```

Comment:
@nimid maybe better to add some prefix name? to avoid duplicate class name (that some 3rd-party modules may have the same name, since it's common name).